src/Types.jl

```diff
@@ -1098,11 +1100,31 @@ function remove_registries(ctx::Context, regs::Vector{RegistrySpec})
     return nothing
 end
 
+function active_pkg_server()
+    server = pkg_server()
+    if server !== nothing
+        request = Downloads.Request(devnull, server, Vector{Pair{String,String}}(undef, 0))
```

Comment:
This is not a stable API and I'm [about to break it](https://github.com/JuliaLang/Downloads.jl/pull/46), so we definitely shouldn't merge before that.

Reply:
I can switch to using a different method. Is there a public API in Downloads that will expose the same info?